charts/datahub/templates/kafka-setup-job.yml

```diff
@@ -74,7 +74,7 @@ spec:
             - name: MAX_MESSAGE_BYTES
               value: {{ . | quote }}
             {{- end }}
-            {{- if eq .Values.global.kafka.schemaregistry.type "INTERNAL" }}
+            {{- if and (eq .Values.global.kafka.schemaregistry.type "INTERNAL") (eq .Values.global.kafka.schemaregistry.type "AWS_GLUE") }}
```

Comment:
I think you wanted an `or` here: https://helm.sh/docs/chart_template_guide/function_list/#or